### PR TITLE
Github Issues

### PR DESCRIPTION
**Pull request by ingalls**
### Context

Create a GH Issue on a PR with an image of the data

**Note**: This is not a replacement for looking at the stats on the job page itself before merging.

Closes: https://github.com/openaddresses/batch/issues/20

### CODE REVIEW

**Comment by iandees** [comment]
> This is not a replacement for looking at the stats on the job page itself before merging.

The comment that gets added should probably highlight any significant changes in the stats. Or at least include a link to the stats page for the run.

**Comment by ingalls** [comment]
Screenshot from 2020-09-14 21-38-27

File: api/lib/ci.js [hunk 1]
```diff
@@ -64,6 +65,73 @@ class CI {
...
+    }
+
+
+    /**
+     * Add an issue showing a preview PNG to a given PR
+     *
+     * @param {Numeric} pr PR Number to add comment to
+     * @param {String} issue Issue body
+     */
+    async add_issue(pr, issue) {
```
Comment (iandees) [inline api/lib/ci.js:110]: This should probably be "comment", not issue.

File: api/lib/ci.js [hunk 2]
```diff
@@ -64,6 +65,73 @@ class CI {
...
+    async format_issue(pool, run) {
+        const jobs = await Run.jobs(pool, run.id);
+        let issue = '';
+
+        for (const job of jobs) {
+            if (!['Warn', 'Success'].includes(job.status)) continue;
+
+            issue = issue + '\n'
+                + `### ${job.source_name}-${job.layer}-${job.name}\n`
+                + `![Preview Image](https://batch.openaddresses.io/api/job/${job.id}/output/source.png)\n`;
```
Comment (iandees) [inline api/lib/ci.js:97]: Can we include a link to the run page for this source here?

**Pull request merged**